cmd/commands/main.go

```diff
@@ -0,0 +1,601 @@
+// Copyright (c) 2013-2017 The btcsuite developers
+// Copyright (c) 2015-2016 The Decred developers
+// Copyright (C) 2015-2022 The Lightning Network Developers
+
+package commands
```

Comment:
Do you want to do a git mv on this file as opposed to the current delete + create?

Reply:
Hmm, I did it that way, but git still wants to see this as a new file... Maybe there are too many changes? Weird.